scenarios/Assistants/assistant-bot-in-a-box/README.md

```diff
@@ -39,7 +39,7 @@ The flow of messages is as follows:
 
 ```
 git clone https://github.com/Azure/AI-in-a-Box
-cd gen-ai/assistant-bot-in-a-box
+cd gen-ai/Assistants/bot-in-a-box
```

Comment:
Is this meant for the current repo and directory paths? e.g.
```suggestion
cd scenarios/Assistants/assistant-bot-in-a-box
```

Reply:
@brandom-msft just for the current repo that uses this git clone https://github.com/Azure/AI-in-a-Box.